website/static/files/build_orca_reconcile.sql

```diff
@@ -6,88 +6,94 @@ SET search_path TO orca, public;
-  COMMENT ON COLUMN reconcile_jobs.orca_archive_location IS 'ORCA S3 Glacier bucket the file resides in.';
+  COMMENT ON COLUMN reconcile_jobs.orca_archive_location IS 'ORCA S3 Glacier bucket the reconciliation targets.';
   COMMENT ON COLUMN reconcile_jobs.status_id IS 'Current status of the job.';
-  COMMENT ON COLUMN reconcile_jobs.inventory_creation_time IS 'Time from the s3 inventory manifest ';
+  COMMENT ON COLUMN reconcile_jobs.inventory_creation_time IS 'Inventory report creation time from the s3 manifest.';
   COMMENT ON COLUMN reconcile_jobs.start_time IS 'Date and time the internal reconcile job started.';
   COMMENT ON COLUMN reconcile_jobs.last_update IS 'Date and time the job status was last updated.';
   COMMENT ON COLUMN reconcile_jobs.end_time IS 'Time the job completed and wrote the report information.';
   COMMENT ON COLUMN reconcile_jobs.error_message IS 'Critical error the job ran into that prevented it from finishing.';
 
 
-  CREATE TABLE reconcile_s3_objects  ( 
-    job_id               	int8 NOT NULL,
-    orca_archive_location	text NOT NULL,
-    key_path             	text NOT NULL,
-    etag                 	text NOT NULL,
-    last_update          	timestamp with time zone NOT NULL,
-    size_in_bytes        	int8 NOT NULL,
-    storage_class        	text NOT NULL,
-    delete_marker        	bool NOT NULL,
-    CONSTRAINT FK_reconcile_jobs_s3_objects FOREIGN KEY(job_id) REFERENCES reconcile_jobs(id)
-    ) PARTITION BY RANGE (orca_archive_location);
+  CREATE TABLE IF NOT EXISTS reconcile_s3_objects  ( 
+    job_id               	int8 NOT NULL
+  , orca_archive_location	text NOT NULL
+  , key_path             	text NOT NULL
+  , etag                 	text NOT NULL
+  , last_update          	timestamp with time zone NOT NULL
+  , size_in_bytes        	int8 NOT NULL
+  , storage_class        	text NOT NULL
+  , delete_marker        	bool NOT NULL
+  , CONSTRAINT FK_reconcile_jobs_s3_objects FOREIGN KEY(job_id) REFERENCES reconcile_jobs(id)
+    ) PARTITION BY LIST (orca_archive_location);
   COMMENT ON TABLE reconcile_s3_objects IS 'Temporary table that holds the listing from the ORCA S3 bucket to use for comparisons against the ORCA catalog.
   Partitions should be placed around orca_archive_location, followed by indexing on key_path once data is ingested.';
   COMMENT ON COLUMN reconcile_s3_objects.job_id IS 'Job the S3 listing is a part of for the comparison. Foreign key to the reconcile jobs table.';
   COMMENT ON COLUMN reconcile_s3_objects.orca_archive_location IS 'ORCA S3 Glacier bucket name where the file is stored.';
-  COMMENT ON COLUMN reconcile_s3_objects.key_path IS 'Full path and file name of the object in the S3 bucket from the S3 list command.';
+  COMMENT ON COLUMN reconcile_s3_objects.key_path IS 'Full path and file name of the object in the S3 bucket.';
   COMMENT ON COLUMN reconcile_s3_objects.etag IS 'AWS etag value from the S3 list command.';
```

Comment:
Need to remove `from the S3 list command` from other comments

Reply:
Fixed